src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -18,8 +18,8 @@ function notMainnet(networkSettings) {
 }
 
 const style = {
-  backgroundColor: AlertUI.primaryColor,
-  color: AlertUI.dark40Color,
+  backgroundColor: AlertUI.lightColor,
```

Comment:
This background color is different than the ones we have nowadays, is that correct?

Reply:
Yes, but by 4 points of light. I will correct it here.

Reply:
Now it is correct, sorry: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/371/commits/716ae6b0dc7aafb4bfe1748394b23ac7a6b38505